iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -465,6 +475,11 @@ public ModuleRegistrationResponse processRegistrationRequest(ModuleRegistrationR
                         .setIsChecked(true)
                         .setDescription("If enabled, mParticle will attempt to coerce string attributes into scalar types (integer, boolean, and float).")
         );
+        eventSettings.add(
```

Comment:
we will add this setting on our side - please remove it from here.

Reply:
Are the name and the values ok?